contrib/helm-chart/templates/nginx-svc.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: nginx
+  annotations:
+    {{- range $key, $value := .Values.nginx.annotations }}
+      {{ $key }}: {{ $value | quote }}
+    {{- end }}
+spec:
+  type: LoadBalancer
```

Comment:
Can we get a NodePort alternative as well? Maybe something like https://github.com/kubernetes/charts/blob/master/stable/postgresql/templates/svc.yaml?